server/api.js

```diff
@@ -95,19 +112,32 @@ const saveUser = (client) => {
 	]);
 };
 
-router.post("/clients", (req, res) => {
-	if (Array.isArray(req.body)) {
-		const promises = req.body.map((client) => {
-			return saveUser(client);
+router.post("/send-messages", (req, res) => {
+	const message = req.body.message;
+	const clientIds = req.body.ids;
+	const timeSent = new Date().toISOString();
+
+	const dbQueries = [];
+	clientIds.forEach((id)=>{
+		console.log("sending SMS to " + id);
+		const uuid = randomUUID();
+		const createQuery = "INSERT INTO messages (client_id, message, successfully_sent, time_sent) VALUES ($1, $2, $3, $4)";
+		dbQueries.push(db.query(createQuery, [id, message, true, timeSent]));
+
+		const updateQuery = "UPDATE clients SET booking_status=2, uuid=$1 WHERE client_id=$2";
+		dbQueries.push(db.query(updateQuery, [uuid, id]));
+	});
+
+	Promise.allSettled(dbQueries)
+  		.then((results) => {
+			let filteredResults = results.filter((result) => result.status === "rejected");
```

Comment:
```suggestion
			const rejectedQueries = results.filter((result) => result.status === "rejected");
```
- We're using `const` by default in this project—let's use it here as we're not changing the variable later
- We could call the variable `rejectedQueries` which better reflects what the variable holds